source/org/zfin/search/service/RelatedDataService.java

```diff
@@ -207,6 +207,15 @@ public List<String> getRelatedDataLinks(SearchResult result) {
         return links;
     }
 
+    private String makeGenomeBrowserLinkByID(String id) {
+        String url = GenomeLocation.Source.ZFIN_Zv9.getUrl() + id;
```

Comment:
Is it always Zv9 assembly?

Reply:
Good question. In this case, I'm preserving the original logic. In the original version, it was only Zv9. So as long as the original version is correct, this one should be. I could make it more explicit by renaming the method to "makeZv9GenomeBrowserLinkByID".

You can see the original version on lines 100 and 111:
```
gBrowseLink = makeLink(GENOME_BROWSER, "/" + ZfinPropertiesEnum.GBROWSE_ZV9_PATH_FROM_ROOT + "?name=" + id);
```